docs/source/gpkit101.rst

```diff
@@ -5,14 +5,13 @@ GPkit is a Python package for defining and manipulating
 geometric programming (GP) models,
 abstracting away the backend solver.
 
-The primary goal of GPkit is to make it easy to create share and explore Geometric Programming models.
-Being fast and mathematically correct tend to align well with this goal.
+The goal of GPkit is to make it easy to create share and explore Geometric Programming models, which tends to align well with being fast and mathematically correct.
```

Comment:
drop second half of this sentence, after the comma (or reword the part about "mathematically correct")?